easybuild/easyblocks/s/star_ccm.py

```diff
@@ -69,15 +94,33 @@ def install_step(self):
 
         env.setvar('IATEMPDIR', tempfile.mkdtemp())
 
-        cmd = ' '.join([
-            self.cfg['preinstallopts'],
-            install_script,
-            "-i silent",
-            "-DINSTALLDIR=%s" % self.installdir,
-            "-DINSTALLFLEX=false",
-            "-DADDSYSTEMPATH=false",
-            self.cfg['installopts'],
-        ])
+        # argument -DINSTALLFLEX is -DINSTALL_LICENSING for the .aol installer
+        if self.aol_install:
+            cmd = ' '.join([
+                self.cfg['preinstallopts'],
+                # The install_script installs also the Siemens Installer Program (SIP)
+                # under $HOME, this is not need to run STAR-CCM+
+                'HOME=%s' % self.builddir,
+                install_script,
+                "-i silent",
+                # for some reason the installation directory's name cannot be the version
+                # So using builddir and then moving to installdir
+                "-DINSTALLDIR=%s" % self.builddir,
+                "-DINSTALL_LICENSING=false",
+                "-DADDSYSTEMPATH=false",
+                self.cfg['installopts'],
+                "&& mv %s/%s %s" % (self.builddir, self.version, self.installdir),
```

Comment:
for the `-r8` variant, suffix `-R8` is added to this directory. simplest solution is to add a wildcard:
```suggestion
                "&& mv %s/%s* %s" % (self.builddir, self.version, self.installdir),
```